include/tvSettings.h

```diff
@@ -2622,6 +2622,79 @@ tvError_t GetLdimZoneShortCircuitStatus(unsigned char* shortcircuit_zone_list, u
  */
 tvError_t GetNumberOfDimmingZones(unsigned int* number_of_dimming_zones);
 
+/**
+ * @brief Sets 2Point Custom Whitebalance
+ *
+ * This function sets Whitebalance for a current picture mode index, primary video source @n
```

Comment:
In the new HAL APIs being define for multi-point WB (and others), we have stopped using the Set and Save pair of functions in favour of a single Set with contextual parameters for picture mode, colour temp, video format and video source.

Ref:
https://www.stb.bskyb.com/confluence/display/2016/EntOS+Video+Settings#EntOSVideoSettings-RDK-VLegacyHAL-tvSettings.7

If you are reworking the 2pt WB APIs, then it's probably best to follow this new style.  Ram and Utkarsh both agreed with this new approach.

A new caps function which exposes the platform ranges for gain+offset and colours would also be highly desirable.
While the INI file has ranges for these available, the HAL API has fixed these:

`* @param[in] value                 - The WhiteBalance Value to be set.Valid range gain (0 - 2047) and offset (-1024 to 1023)`

Ideally the caps function should expose these ranges.